flink-ml-iteration/src/main/java/org/apache/flink/iteration/operator/OperatorUtils.java

```diff
@@ -104,6 +107,33 @@ public static StreamConfig createWrappedOperatorConfig(
             }
         }
 
+        StreamConfig.InputConfig[] inputs = wrapperConfig.getInputs(cl);
+        for (int i = 0; i < inputs.length; ++i) {
+            if (inputs[i] instanceof NetworkInputConfig) {
+                TypeSerializer<?> typeSerializerIn =
+                        ((NetworkInputConfig) inputs[i]).getTypeSerializer();
+                if (typeSerializerIn instanceof IterationRecordSerializer) {
```

Comment:
It seems that `typeSerializerIn` and `typeSerializerOut` should always be `IterationRecordSerializer` for wrapped iteration operators if they are not null.

Is there any case that this condition does not hold?

Reply:
Yes, you're right. I removed the  `if ... instanceOf ...` check. And seems that we did not modify the `StateKeySerializer` at all, so I also removed this part.

Reply:
Yes, the `typeSerializerIn` and `typeSerializerOut` should be `IterationRecordSerializer` in the current usage. But since we are providing a public utility method, I think we'd better check if the `serializer`s are not null and an instance of `IterationRecordSerializer`.